aries-test-harness/features/didcomm-v2.feature

```diff
@@ -0,0 +1,48 @@
+@DIDComm-V2 @UsesCustomParameters
+Feature: DIDComm V2 Establishing Connections (with Issue Credential example)
+
+   @T001-DIDComm-V2
+   Scenario: Establish a connection between two agents using DIDComm V2
+      Given we have "2" agents
+         | name | role      |
+         | Acme | responder |
+         | Bob  | requester |
+
+      And "Acme" is running with parameters "{"mime-type":["didcomm/v2"]}"
+      And "Bob" is running with parameters "{"mime-type":["didcomm/v2"]}"
+
+      When "Acme" creates a didcomm v2 invitation
+      And "Bob" accepts the didcomm v2 invitation from "Acme"
+
+      # Bob must send a message to acme
+      When "Bob" requests mediation from "Acme"
+
+      And "Acme" has received a didcomm v2 message from "Bob" and created a connection
```

Comment:
Preferably, would have a step after this, to verify that they both have a connection. The standard step ("Acme" and "Bob" have a connection) expects a protocol-complete webhook message, for either didexchange or connection-protocol, and didcomm v2 doesn't work that way.

Reply:
Could you not just create a new step phrased a little differently and implemented it in didcimm-v2.py? 
like 
`"Acme" and "Bob" have a didcomm-v2 connection`
as you did with `When "Acme" creates a didcomm v2 invitation`

Reply:
No, it's not possible to combine things completely - if Acme creates a DIDComm V2 invitation, and Bob accepts that invitation, Acme won't know about Bob until Bob sends Acme a message. The first message Bob sends to Acme isn't part of a standard start-connection protocol (like didexchange is), it can be the start of any protocol that's supported by DIDComm V2 (like issue-credential or present-proof V3). So I needed an additional step for Acme to report its new connection to the test harness, after Acme receives its first message from Bob.

I can certainly combine the create-invitation and accept-invitation steps, though!